### DISCUSSION TITLE
Can't upload proof of school in application

### DISCUSSION THREAD

**Discussion by lukelehan** [community/community#58492] [GitHub Education]
### Hi everyone,

I can't upload my proof as it says it can't find my school name on the proof. It's either because my school is listed with it's full name on GitHub and all the emails and proof I have use a shortened version, or it's unreadable because GitHub's camera capture isn't the same quality as my camera app for some reason. Can anyone help me?

**Comment by the-officialjosh** [comment]
SInce you are at school you should try getting your transcript or something of the sort from your school records department as the said setting is automated so try to get the one that corresponds to what your school provided to the platform

**Comment by samus-aran** [comment]
Hi @circuitdx 👋 

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. We would also highlight that School logos are not detectable, if you can write your school's name and submit it alongside the logo that would help with the detection process.

Hope this helps 👍
